api/lib/domain/usecases/correct-answer-then-update-assessment.js

```diff
@@ -100,7 +100,7 @@ function _getSkillsFilteredByStatus(knowledgeElements, targetSkills, status) {
   return knowledgeElements
     .filter((knowledgeElement) => knowledgeElement.status === status)
     .map((knowledgeElement) => knowledgeElement.skillId)
-    .filter((skillId) => targetSkills.find((skill) => skill.id === skillId));
+    .map((skillId) => targetSkills.find((skill) => skill.id === skillId));
```

Comment:
Je suis pas un très grand fan de `lodash` mais j'ai vu que vous l'utilisez beaucoup. Ici on pourrait faire (si je dis pas de bêtises) : 
```suggestion
    .map((skillId) => _.find(targetSkills, { id: skillId }));
```

Reply:
Effectivement le `_.find` ici permettrait de simplifier un peu l'expression, mais la puissance de lodash se déploie surtout quand on exprime les traitements souhaités à un plus haut niveau. Ici par exemple toute la séquence `filter`+`map`+`map(find)` pourrait probablement être remplacée par :

```js
function _getSkillsFilteredByStatus(knowledgeElements, targetSkills, status) {
	return _.intersectionWith(
		targetSkills,
		_.filter(knowledgeElements, { status }),
	 	({ id }, { skillId }) => id == skillId
	);
}
```

Reply:
Il y a surement des améliorations a faire. Vu le bug, je préfère revenir en arrière pour être sûr d'avoir le comportement précédent.